modules/misc/xdg-autostart.nix

```
@@ -35,6 +41,9 @@ in {
   };
 
   config = mkIf (cfg.enable && cfg.entries != [ ]) {
-    xdg.configFile = listToAttrs (map mapDesktopEntry cfg.entries);
+    xdg.configFile.autostart = {
```

nit: I personally like to emphasize attrset keys in NixOS options by passing them in quotes. This is not a blocker for merging though.

```suggestion
    xdg.configFile."autostart" = {
```